EIPS/eip-5792.md

```diff
@@ -207,6 +216,26 @@ As with the return value of `wallet_sendCalls`, the batch identifier may be any
 }
 ```
 
+### `wallet_showCallsStatus`
```

Comment:
what's the motivation for this one?

Reply:
this was in the original 5792. more advanced wallets (eg 4337) will have the most context on tx statuses (such as pending in bundler mempool). could also do things like speeding up / replacing transactions here

Reply:
@arein idea A simple motivating case would be a user wants to speed up or drop a transaction, and the app wants to help kick that off, this would be a screen where a wallet could provide those actions. 

More generally this is a place where a wallet could show a user a lot more details about what happened, if they want to see it.